src/Tribe/Editor/Provider.php

```diff
@@ -126,10 +126,21 @@ protected function hook() {
 			tribe_callback( 'tickets.editor.blocks.attendees', 'register' )
 		);
 
-		add_action(
-			'block_categories',
-			tribe_callback( 'tickets.editor', 'block_categories' )
-		);
+		global $wp_version;
+		if( version_compare( $wp_version, '5.8', '<' ) ) {
+			// WP version is less then 5.8.
+			add_action(
+				'block_categories',
+				tribe_callback( 'tickets.editor', 'block_categories' )
+			);
+		} else {
+			// WP version is 5.8 or above.
+			add_action(
+				'block_categories_all',
+				tribe_callback( 'tickets.editor', 'block_categories' )
+			);
+		}
```

Comment:
might be able to simplify this by doing:

```
$block_categories_action = version_compare( $wp_version, '5.8', '<' ) ? 'block_categories' : 'block_categories_all';
add_action(
	$block_categories_action,
	tribe_callback( 'tickets.editor', 'block_categories' )
);
```

Reply:
@codingmusician comment on this?

Reply:
@paulmskim Sorry. My comment went into "Pending" status for some reason. LOL! See above.

Reply:
Sounds good, just a preference.